framework/src/bundle/BundlePrivate.cpp

```diff
@@ -420,6 +420,10 @@ namespace cppmicroservices
     {
         auto l = this->Lock();
         US_UNUSED(l);
+        if(coreCtx->FrameworkStopped()){
+            throw std::runtime_error("Bundle #" + util::ToString(id) + " (location=" + location
```

Comment:
I find that logging the bundle's symbolic name is more useful than the bundle id.

Reply:
fixed. was following other throw's within this file, but have updated to use symbolic name. May be worth converting all throws to use one or the other at some point

Reply:
I agree its worth converting other throw messages. If its not too much effort, can you convert the other throw messages in this file? Thanks.